app/models/note.rb

```diff
@@ -21,9 +21,10 @@ class Note < ApplicationRecord
   enum note_type: { review: 0, critique: 1 }
 
   def content_length
-    return 'short' if word_count.between(0, short_threshold)
-    return 'medium' if word_count.between(short_threshold + 1, medium_threshold)
-    'long'
+    length_by_word_count = { 0..short_threshold => 'short',
+                             (short_threshold + 1)..medium_threshold => 'medium',
+                             (medium_threshold + 1).. => 'long' }
+    length_by_word_count.select { |count_range| count_range.include? word_count }.values.first
```

Comment:
```suggestion
    length_by_word_count[word_count]
```
No funciona así? Que así ya diría que es menos eficiente que los ifs anidados.